enclave-runtime/Cargo.toml

```diff
@@ -74,6 +75,8 @@ jsonrpc-core = { default-features = false, git = "https://github.com/scs/jsonrpc
 # mesalock
 env_logger = { git = "https://github.com/integritee-network/env_logger-sgx" }
 log = { git = "https://github.com/integritee-network/log-sgx" }
+# Todo: use the `once_cell` included in rusts core library once we use rust v1.70.0
```

Comment:
How about adding the issues here as well?
https://github.com/integritee-network/worker/issues/1313